<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Alvaro de Miguel</author>
 <class>ActividadBase</class>
 <widget class="QWidget" name="ActividadBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>516</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nueva Actividad</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../pluginbf_profesor/gestionaula.qrc">
    <normaloff>:/ImgGestionAula/icons/actividad.png</normaloff>:/ImgGestionAula/icons/actividad.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../pluginbf_profesor/gestionaula.qrc">:/ImgGestionAula/icons/actividad.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <widget class="QToolButton" name="mui_guardar">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Guardar actividad</string>
          </property>
          <property name="whatsThis">
           <string>Guardar actividad</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../bulmages.qrc">
            <normaloff>:/Images/edit-save.png</normaloff>:/Images/edit-save.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="shortcut">
           <string extracomment="Guardar">Ctrl+S</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_borrar">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Borrar actividad</string>
          </property>
          <property name="whatsThis">
           <string>Borrar actividad</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../bulmages.qrc">
            <normaloff>:/Images/edit-delete.png</normaloff>:/Images/edit-delete.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+D</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_imprimir">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Imprimir actividad</string>
          </property>
          <property name="whatsThis">
           <string>Imprimir actividad</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../bulmages.qrc">
            <normaloff>:/Images/document-print.png</normaloff>:/Images/document-print.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+I</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>193</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="mui_plugbotones">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>41</width>
            <height>34</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="mui_tab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Tab1">
      <attribute name="title">
       <string>&amp;Datos generales</string>
      </attribute>
      <layout class="QGridLayout" name="_2">
       <property name="margin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="textLabel2_5">
         <property name="text">
          <string>Codigo Actividad</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_codigoactividad</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="mui_codigoactividad">
         <property name="maxLength">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="textLabel2_2">
         <property name="text">
          <string>Nombre completo:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_nombreactividad</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QLineEdit" name="mui_nombreactividad">
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="textLabel2_2_2">
         <property name="text">
          <string>Lugar</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_corpcliente</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QLineEdit" name="mui_corpcliente">
         <property name="maxLength">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="textLabel2_6">
         <property name="text">
          <string>Direccion:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>mui_dircliente</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QLineEdit" name="mui_dircliente">
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="12" column="1" colspan="2">
        <widget class="BlSearchWidget" name="mui_idprofesor" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>100</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="mui_vigenteactividad">
         <property name="text">
          <string>Vigente</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="BlComboBox" name="mui_idtipoactividad">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Tipo de Actividad</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="mui_precioactividad"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Precio Cuota</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="BlDateSearch" name="mui_finicialactividad"/>
       </item>
       <item row="9" column="1">
        <widget class="BlDateSearch" name="mui_ffinalactividad"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Fecha Inicial</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Fecha Final</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2" rowspan="5">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Sesiones Semanales</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QCheckBox" name="mui_lunesactividad">
            <property name="text">
             <string>Lunes</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="mui_martesactividad">
            <property name="text">
             <string>Martes</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="mui_miercolesactividad">
            <property name="text">
             <string>Miercoles</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="mui_sabadoactividad">
            <property name="text">
             <string>Sabado</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="mui_domingoactividad">
            <property name="text">
             <string>Domingo</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="mui_viernesactividad">
            <property name="text">
             <string>Viernes</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="mui_juevesactividad">
            <property name="text">
             <string>Jueves</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Periodicidad Cuota</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="BlPeriodicityComboBox" name="mui_periodoactividad">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Hora Inicio</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="mui_hinicialactividad"/>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Duraci√≥n</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="mui_duracionactividad"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOtros">
      <attribute name="title">
       <string>Otros datos</string>
      </attribute>
      <layout class="QGridLayout" name="_3">
       <property name="margin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="textLabel2_4_2_2">
         <property name="text">
          <string>Comentarios:</string>
         </property>
         <property name="buddy">
          <cstring>mui_comentariosactividad</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTextEdit" name="mui_comentariosactividad">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Alumnos</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="ListAlumnosActividadView" name="mui_alumnosList"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Asistencias</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="ListFaltasAsistenciaActividadView" name="mui_faltasAsistenciaList"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>201</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mui_aceptar">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mui_cancelar">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="3"/>
 <customwidgets>
  <customwidget>
   <class>BlDateSearch</class>
   <extends>QLineEdit</extends>
   <header>bldatesearch.h</header>
  </customwidget>
  <customwidget>
   <class>ListAlumnosActividadView</class>
   <extends>QTableWidget</extends>
   <header>actividadview.h</header>
  </customwidget>
  <customwidget>
   <class>BlSearchWidget</class>
   <extends>QWidget</extends>
   <header>blsearchwidget.h</header>
  </customwidget>
  <customwidget>
   <class>BlComboBox</class>
   <extends>QComboBox</extends>
   <header>blcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>ListFaltasAsistenciaActividadView</class>
   <extends>QTableWidget</extends>
   <header>actividadview.h</header>
  </customwidget>
  <customwidget>
   <class>BlPeriodicityComboBox</class>
   <extends>QComboBox</extends>
   <header>blperiodicitycombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mui_tab</tabstop>
  <tabstop>mui_codigoactividad</tabstop>
  <tabstop>mui_nombreactividad</tabstop>
  <tabstop>mui_corpcliente</tabstop>
  <tabstop>mui_dircliente</tabstop>
  <tabstop>mui_vigenteactividad</tabstop>
  <tabstop>mui_idtipoactividad</tabstop>
  <tabstop>mui_precioactividad</tabstop>
  <tabstop>mui_periodoactividad</tabstop>
  <tabstop>mui_finicialactividad</tabstop>
  <tabstop>mui_ffinalactividad</tabstop>
  <tabstop>mui_idprofesor</tabstop>
  <tabstop>mui_lunesactividad</tabstop>
  <tabstop>mui_martesactividad</tabstop>
  <tabstop>mui_miercolesactividad</tabstop>
  <tabstop>mui_juevesactividad</tabstop>
  <tabstop>mui_viernesactividad</tabstop>
  <tabstop>mui_sabadoactividad</tabstop>
  <tabstop>mui_domingoactividad</tabstop>
  <tabstop>mui_aceptar</tabstop>
  <tabstop>mui_cancelar</tabstop>
  <tabstop>mui_faltasAsistenciaList</tabstop>
  <tabstop>mui_comentariosactividad</tabstop>
  <tabstop>mui_alumnosList</tabstop>
 </tabstops>
 <resources>
  <include location="../../../bulmages.qrc"/>
  <include location="../pluginbf_profesor/gestionaula.qrc"/>
 </resources>
 <connections/>
</ui>
